### DISCUSSION TITLE
Global area objects config

### DISCUSSION THREAD

**Discussion by GuyZipory** [dwainscheeren/dwains-lovelace-dashboard#342] [Q&A]
Hey,
how do I config these two:
![image](https://user-images.githubusercontent.com/11477714/145453278-9aa510f0-2f45-48f3-a4d3-9777ce5abb24.png)

I didn't manage to find that on the docs.

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
